cyclops-ui/src/components/pages/NodeDetails.tsx

```diff
@@ -190,6 +191,97 @@ const NodeDetails = () => {
       ),
   });
 
+  const getColumnFilterProps = (
+    dataIndex: DataIndex,
+  ): ColumnType<DataSourceType> => ({
+    filterDropdown: ({
+      setSelectedKeys,
+      selectedKeys,
+      confirm,
+      clearFilters,
+      close,
+    }) => {
+      const uniqueValues = Array.from(
+        new Set(node.pods.map((pod: DataSourceType) => pod[dataIndex])),
+      );
+
+      return (
+        <div
+          style={{
+            padding: 8,
+            maxHeight: 200,
+            overflowY: "auto",
+            margin: "8px 0",
+            border: "1px solid #ccc",
+          }}
+        >
+          <div style={{ marginBottom: 8 }}>
+            {uniqueValues.map((value) => (
+              <div
+                key={value}
+                style={{ display: "flex", alignItems: "center" }}
+              >
+                <Checkbox
+                  checked={selectedKeys.includes(value)}
+                  onChange={(e) => {
+                    const checked = e.target.checked;
+                    const updatedKeys = checked
+                      ? [...selectedKeys, value]
+                      : selectedKeys.filter((v: any) => v !== value);
+                    setSelectedKeys(updatedKeys);
+                  }}
+                />
+                <span style={{ marginLeft: 8 }}>{value}</span>
+              </div>
+            ))}
+          </div>
+          <Space>
+            <Button
+              type="primary"
+              icon={<SearchOutlined />}
+              size="small"
+              style={{ width: 90 }}
+              onClick={() =>
+                handleSearch(selectedKeys as string[], confirm, dataIndex)
+              }
+            >
+              Filter
+            </Button>
+            <Button
+              onClick={() => clearFilters && handleReset(clearFilters, confirm)}
+              size="small"
+              style={{ width: 90 }}
+            >
+              Reset
+            </Button>
+            <Button
+              type="link"
+              size="small"
+              onClick={() => {
+                close();
+              }}
+            >
+              Close
+            </Button>
+          </Space>
+        </div>
+      );
+    },
+    filterIcon: (filtered: boolean) => (
+      <SearchOutlined style={{ color: filtered ? "#1677ff" : undefined }} />
```

Comment:
Can you set `#ff8803` here?  Its Cyclops orange 😄 

I see that we left the same blue on line 169, so could you also update that?

Reply:
Ive changed it back to orange... honestly have no idea why i changed it to that blue color, i must of used it on a different file and copied it or something.